auslib/dockerflow.py

```diff
@@ -26,8 +26,11 @@ def heartbeat():
         """Per the Dockerflow spec:
         Respond to /__heartbeat__ with a HTTP 200 or 5xx on error. This should
         depend on services like the database to also ensure they are healthy."""
-        database_entry_value = heartbeat_database_fn(dbo)
-        return Response(str(database_entry_value), headers={"Cache-Control": "no-cache"})
+        try:
+            database_entry_value = heartbeat_database_fn(dbo)
+            return Response(str(database_entry_value), headers={"Cache-Control": "no-cache"})
```

Comment:
This Cache-Control should be set to the same value as below. Using no-cache here means we're still vulnerable to DoS when the database is up. Other than that, this looks good to me!